example/params.har

```diff
@@ -1,27 +1,42 @@
 {
-    "log": {
-        "entries": [
-            {
-                "index": 0,
-                "request": {
-                    "method": "POST",
-                    "url": "http://test.loadimpact.com/path",
-                    "httpVersion": "HTTP/1.1",
-                    "postData": {
-                        "mimeType": "application/x-www-form-urlencoded",
-                        "params": [
-                            {
-                                "name": "login",
-                                "value": "admin"
-                            },
-                            {
-                                "name": "password",
-                                "value": "123"
-                            }
-                        ]
-                    }
-                }
-            }
-        ]
-    }
+  "log": {
+    "entries": [
+      {
+        "variables": [
+          {
+            "name": "userName",
+            "type": 0,
+            "expression": "user.name"
+          }
+        ],
+        "request": {
+          "method": "GET",
+          "url": "http://test.loadimpact.com/path"
+        }
+      },
+      {
+        "request": {
+          "method": "POST",
+          "url": "http://test.loadimpact.com/path",
+          "postData": {
+            "mimeType": "application/x-www-form-urlencoded",
+            "params": [
+              {
+                "name": "login",
+                "value": "admin"
+              },
+              {
+                "name": "password",
+                "value": ""
+              },
+              {
+                "name": "${userName}",
```

Comment:
Shouldn't the param-name be hardcoded, and the variable be in the `value`-property?

Reply:
It is possible to use variables dynamically in urls, headers, queryParams etc. I added this to the example har just to see if the fix was still working for resolved cases of params.

Reply:
I can add the example for dynamic value as well 👍

Reply:
Ah, then it was just the fact that the variable was called `userName` which threw me off.